### PR TITLE
Add times, dates and stations datasets

### PR DESCRIPTION
**Pull request by saattrupdan**
This PR adds the following:

1. A dates dataset, with day, month and year
2. A times dataset, with hour and minute, as well as expressions such as "kvart i fire"
3. A script that interleaves them according to given sampling probabilities
4. Setup of docker - running `make docker` will build the dataset in the `data` folder.
5. Updates the readme with a quick start

### CODE REVIEW

File: src/tts_text/dates.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "Maj",
+    "Juni",
+    "Juli",
+    "August",
+    "September",
+    "Oktober",
+    "November",
+    "December",
+]
+YEARS = [year for year in range(2000, 2030)]
```
Comment (AJDERS) [inline src/tts_text/dates.py]: Maybe also include a few years before 2000?

**Review by AJDERS — CHANGES_REQUESTED** [review]
Almost LGTM, minor change

File: src/tts_text/dates.py (hunk 1 above)
Comment (saattrupdan) [reply src/tts_text/dates.py]: Added 1970-1999 as well now

**Review by AJDERS — APPROVED** [review]
LGTM!

**Pull request merged**